subprojects/gradle/cd/build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
     id("convention.kotlin-jvm")
-    id("convention.publish-gradle-plugin")
+    id("convention.publish-gradle-plugin-legacy")
```

Comment:
I don't get what is the purpose of that plugin?

Reply:
convention for publication of gradle plugins;

problem is here: https://github.com/avito-tech/avito-android/pull/791/files#diff-e7b95c7356270a4bf48fea167dfbb47a8fa67e7a8ab35930fce58a7e9c047f2cR12

trying to publish new plugin correctly with new convention along with legacy one